libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts

```diff
@@ -264,4 +264,23 @@ describe('FormFieldComponent', () => {
       });
     });
   });
+
+  describe('When nested inside a kirby-item', () => {
+    beforeEach(() => {
+      spectator = createHost(
+        `<kirby-item>
+          <kirby-form-field>
+            <input kirby-input />
+          </kirby-form-field>
+        </kirby-item>`
+      );
+    });
+
+    it('should render with no bottom margin', () => {
+      const formFieldElement = spectator.queryHost('kirby-form-field');
+      expect(formFieldElement).toHaveComputedStyle({
+        'margin-bottom': '0px',
+      });
+    });
+  });
```

Comment:
Add tests for text-align when `kirby-form-field[slot=end]` + font-weight when `&[type='number'] `